extensions/msal-node-extensions/test/util/TestConstants.ts

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AccountInfo, AuthenticationResult } from "@azure/msal-common";
+import { Account, ErrorStatus, MsalRuntimeError } from "@azure/msal-node-runtime";
+
+export const testMsalRuntimeAccount: Account = {
+    accountId: "MTIzLXRlc3QtdWlk.NDU2LXRlc3QtdXRpZA==",
+    homeAccountId: "123-test-uid.456-test-utid",
+    environment: "login.windows.net",
+    realm: "456-test-utid",
+    localAccountId: "123-test-uid",
+    username: "JohnSmith@contoso.com",
+    givenName: "John",
+    familyName: "Smith",
+    middleName: "M",
+    displayName: "John Smith",
+    additionalFieldsJson: "",
+    homeEnvironment: "login.windows.net",
+    clientInfo: "eyJ1aWQiOiIxMjMtdGVzdC11aWQiLCJ1dGlkIjoiNDU2LXRlc3QtdXRpZCJ9"
+}
+
+export const testAccountInfo: AccountInfo = {
+    homeAccountId: testMsalRuntimeAccount.homeAccountId,
+    environment: testMsalRuntimeAccount.environment,
+    tenantId: testMsalRuntimeAccount.realm,
+    username: testMsalRuntimeAccount.username,
+    localAccountId: testMsalRuntimeAccount.localAccountId,
+    idTokenClaims: undefined,
+    name: testMsalRuntimeAccount.displayName,
+    nativeAccountId: testMsalRuntimeAccount.accountId
+};
+
+export const msalRuntimeExampleError: MsalRuntimeError = {
+    errorCode: 0,
+    errorStatus: ErrorStatus.Unexpected,
+    errorContext: "Test Error",
+    errorTag: 0
+};
+
+export const TEST_CLIENT_ID = "0813e1d1-ad72-46a9-8665-399bba48c201";
+
+export const TEST_ID_TOKEN = "eyJ0eXAiOiJKV1QiLCJhbGciOiJSUzI1NiIsImtpZCI6IjFMVE16YWtpaGlSbGFfOHoyQkVKVlhlV01xbyJ9.eyJ2ZXIiOiIyLjAiLCJpc3MiOiJodHRwczovL2xvZ2luLm1pY3Jvc29mdG9ubGluZS5jb20vOTE4ODA0MGQtNmM2Ny00YzViLWIxMTItMzZhMzA0YjY2ZGFkL3YyLjAiLCJzdWIiOiJBQUFBQUFBQUFBQUFBQUFBQUFBQUFJa3pxRlZyU2FTYUZIeTc4MmJidGFRIiwiYXVkIjoiNmNiMDQwMTgtYTNmNS00NmE3LWI5OTUtOTQwYzc4ZjVhZWYzIiwiZXhwIjoxNTM2MzYxNDExLCJpYXQiOjE1MzYyNzQ3MTEsIm5iZiI6MTUzNjI3NDcxMSwibmFtZSI6IkFiZSBMaW5jb2xuIiwicHJlZmVycmVkX3VzZXJuYW1lIjoiQWJlTGlAbWljcm9zb2Z0LmNvbSIsIm9pZCI6IjAwMDAwMDAwLTAwMDAtMDAwMC02NmYzLTMzMzJlY2E3ZWE4MSIsInRpZCI6IjMzMzgwNDBkLTZjNjctNGM1Yi1iMTEyLTM2YTMwNGI2NmRhZCIsIm5vbmNlIjoiMTIzNTIzIiwiYWlvIjoiRGYyVVZYTDFpeCFsTUNXTVNPSkJjRmF0emNHZnZGR2hqS3Y4cTVnMHg3MzJkUjVNQjVCaXN2R1FPN1lXQnlqZDhpUURMcSFlR2JJRGFreXA1bW5PcmNkcUhlWVNubHRlcFFtUnA2QUlaOGpZIn0=.1AFWW-Ck5nROwSlltm7GzZvDwUkqvhSQpm55TQsmVo9Y59cLhRXpvB8n-55HCr9Z6G_31_UbeUkoz612I2j_Sm9FFShSDDjoaLQr54CreGIJvjtmS3EkK9a7SJBbcpL1MpUtlfygow39tFjY7EVNW9plWUvRrTgVk7lYLprvfzw-CIqw3gHC-T7IK_m_xkr08INERBtaecwhTeN4chPC4W3jdmw_lIxzC48YoQ0dB1L9-ImX98Egypfrlbm0IBL5spFzL6JDZIRRJOu8vecJvj1mq-IUhGt0MacxX8jdxYLP-KUu2d9MbNKpCKJuZ7p8gwTL5B7NlUdh_dmSviPWrw";
```

Comment:
Does this not flag? I thought we are supposed to use only `dummy` creds?

Reply:
This is a dummy cred and has been marked to be ignored in the credscan tooling